examples/kubernetes/env.sh

```diff
@@ -6,16 +6,18 @@
 # use cases just need KUBECTL=kubectl, we'll make that the default.
 KUBECTL=${KUBECTL:-kubectl}
 
-# Kubernetes api address for $KUBECTL 
-# The default value is 127.0.0.1:8080
-# When the Kubernetes api server is not local, We can easily access the api by edit KUBERNETES_API_SERVER's value
-KUBERNETES_API_SERVER=${KUBERNETES_API_SERVER:-'127.0.0.1:8080'}
+# Kubernetes api address for $KUBECTL. When the Kubernetes api server is not
```

Comment:
nit: s/api/API/g

That's also how they call it in the docs: https://kubernetes.io/docs/admin/kube-apiserver/

> The Kubernetes API server ...